### PR TITLE
Render limits

### PR DESCRIPTION
**Pull request by rochoa**

### CODE REVIEW

**Comment by rochoa** [comment]
cc @javisantana @zenitraM

File: lib/cartodb/controllers/template_maps.js [hunk 1]
```diff
@@ -408,7 +408,8 @@ TemplateMapsController.prototype.instantiateTemplate = function(req, res, templa
             if ( req.profiler ) req.profiler.done('TemplateMaps_instance');
             if ( err ) throw err;
             layergroup = instance;
-            fakereq = { query: {}, params: {}, headers: _.clone(req.headers),
+            fakereq = {query: {}, params: {}, headers: _.clone(req.headers),
```
Comment (javisantana) [inline lib/cartodb/controllers/template_maps.js]: just to be sure, we added user here for the url_rewrite ?

**Comment by javisantana** [comment]
:es only one question just to be sure

File: lib/cartodb/controllers/template_maps.js (hunk 1 above)
Comment (rochoa) [inline lib/cartodb/controllers/template_maps.js]: Nop, there is a ticket for that: https://github.com/CartoDB/Windshaft-cartodb/issues/280

I will add it there and make it part of the next version ;-)

**Comment by zenitraM** [comment]
:+1:

**Pull request merged**